pkg/dryrun/dryrun_utils.go

```diff
@@ -55,9 +59,9 @@ func compareStatus(cmd *cobra.Command,
 	isStatusMatch := compareStatusObj(cmd, true, "", inputStatus, resultStatus, noColor)
 
 	if isStatusMatch {
-		cmd.Println(successColor("\033[1m Expected status matches the actual status \033[0m", noColor))
+		cmd.Println(successColor(boldColor(" Expected status matches the actual status", noColor), noColor))
```

Comment:
I removed the trailing space here. I can put it back if there was a reason for it, but I think I'd generally prefer to avoid trailing spaces...